#! /bin/sh /usr/share/dpatch/dpatch-run
## cfg.cfg-debianize.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debianize cgi.cfg

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' pkg-icinga~/sample-config/cgi.cfg.in pkg-icinga/sample-config/cgi.cfg.in
--- pkg-icinga~/sample-config/cgi.cfg.in	2012-05-11 22:33:54.000000000 +0200
+++ pkg-icinga/sample-config/cgi.cfg.in	2012-06-01 08:42:30.385584123 +0200
@@ -56,7 +56,22 @@
 
 http_charset=utf-8
 
+# NAGIOS PROCESS CHECK COMMAND
+# This is the full path and filename of the program used to check
+# the status of the Nagios process.  It is used only by the CGIs
+# and is completely optional.  However, if you don't use it, you'll
+# see warning messages in the CGIs about the Nagios process
+# not running and you won't be able to execute any commands from
+# the web interface.  The program should follow the same rules
+# as plugins; the return codes are the same as for the plugins,
+# it should have timeout protection, it should output something
+# to STDIO, etc.
+#
+# Note: The command line for the check_nagios plugin below may
+# have to be tweaked a bit, as different versions of the plugin
+# use different command line arguments/syntaxes.
 
+icinga_check_command=/usr/lib/nagios/plugins/check_nagios /var/lib/icinga/status.dat 5 '/usr/sbin/icinga'
 
 # CONTEXT-SENSITIVE HELP
 # This option determines whether or not a context-sensitive
